<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubPanel</class>
 <widget class="QWidget" name="SubPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>232</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>232</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>232</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(85, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="curveLineBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>曲 线</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="lineBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>直 线</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="triangleBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>三角形</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="circleBt">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>圆/椭圆</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="rectBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>矩 形</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="diamondBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>菱 形</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="trapeziaBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>梯 形</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="parallelBt">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:4px groove rgb(255, 165, 0);
	border-radius:16px;
	font: 75 11pt &quot;微软雅黑&quot;;

            }          
QPushButton:hover{ 
	color: rgb(0, 255, 255);
border:4px groove rgb(0, 255, 255);
            } 
            
QPushButton:pressed{ 
	color: rgb(255, 85, 0);
border:4px groove rgb(255, 85, 0);
            }</string>
      </property>
      <property name="text">
       <string>四边形</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>300</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>180</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(85, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QPushButton" name="colorBt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>32</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background-image: url(:/resource/rgb32.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="fillBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>填充</string>
    </property>
   </widget>
   <widget class="QSlider" name="penWidth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>240</width>
      <height>36</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>36</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
		border-radius: 5px;
        height: 10px;
}
QSlider::handle:horizontal {
        width: 32px;
		background:transparent;
	    background-image: url(:/resource/handle.png);
        margin: -13px -1px -9px 1px;
}
QSlider::sub-page:horizontal{    
		background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(72, 71, 71), stop: 1 rgb(255, 0, 0));
}

QSlider::add-page:horizontal {
   		 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(124, 124, 124), stop: 1.0 rgb(72, 71, 71));
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>34</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>34</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>34</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);
font: 75 15pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="dashDotBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>83</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>点画线</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dashBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>虚线</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dotBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>点线</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lineTypeTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);
font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>绘制类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="penTypeTitle">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>设置画笔</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
